ethcore/src/miner/miner.rs

```diff
@@ -1123,6 +1123,9 @@ impl MinerService for Miner {
 		// First update gas limit in transaction queue
 		self.update_gas_limit(chain);
 
+		// Update minimal gas price
+		self.recalibrate_minimal_gas_price();
```

Comment:
wouldn't this lead to sending off an HTTP request every time `chain_new_blocks` is called?